runelite-api/src/main/java/net/runelite/api/IconID.java

```diff
@@ -43,12 +48,8 @@
 	NO_ENTRY(11),
 	CHAIN_LINK(12);
 
-	int index;
-
-	IconID(int index)
-	{
-		this.index = index;
-	}
+	@Getter

```

Comment:
You shouldn't need this annotation if `@Getter` is already on the class.

Reply:
Ah, true. Removed